docs/openapi.json

```diff
@@ -35,6 +35,16 @@
                             }
                         }
                     },
+                    "500": {
+                        "description": "Query can not be validated, LLM is not accessible or other internal error",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/ErrorResponse"
+                                }
+                            }
+                        }
+                    },
                     "422": {
                         "description": "Validation Error",
```

Comment:
Does this mean the query content is invalid but its format is good?

Reply:
query content from LLM point of view. It's user query, not anything related to REST API, if your question is about it